changelogs/CHANGELOG-1.4.md

```diff
@@ -17,6 +17,11 @@ https://velero.io/docs/v1.4/upgrade-to-1.4/
   * Adjust restic default time out to 4 hours and base pod resource requests to 500m CPU/512Mi memory. (#2696, @nrb)
   * capture version of the CRD prior before invoking the remap_crd_version backup item action (#2683, @ashish-amarnath)
 
+
+## v1.4.1
```

Comment:
1.4.2

Reply:
Read the section :)

Reply:
I missed that :) Thanks for making this clear.